src/config/constants.ts

```diff
@@ -794,10 +793,10 @@ export const DOCKER_FILE_ERROR_MESSAGE = 'Unable to locate Dockerfile as source
 export const DEFAULT_GIT_BRANCH_VALUE = '--'
 export const SOURCE_NOT_CONFIGURED_MESSAGE =
     'Source is not configured for one or more git repositories. Please configure and try again.'
-
+export const NO_COMMIT_SELECTED = 'No commit is selected'
```

Comment:
Please move it to constant messaging

Reply:
We will be merging both in next tech debt iteration